kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/JbpmKieServerBaseIntegrationTest.java

```diff
@@ -75,6 +75,7 @@
     protected static final String PROCESS_ID_USERTASK_WITH_SLA = "definition-project.UserTaskWithSLA";
     protected static final String PROCESS_ID_USERTASK_WITH_SLA_ON_TASK = "definition-project.UserTaskWithSLAOnTask";
     protected static final String PROCESS_ID_USERTASK_WITH_ROLLBACK = "UserTaskWithRollback";
+    protected static final String PROCESS_ID_USERTASK_DIFF_POTUSERS = "definition-project.usertask-diff-potusers";
```

Comment:
this is going to make fail other tests because they are counting the number of process definitions.. Create a new project.